src/expression/types/color.ts

```diff
@@ -56,10 +56,10 @@ class Color {
         }
     }
 
-    static black: Color;
-    static white: Color;
-    static transparent: Color;
-    static red: Color;
+    static black = new Color(0, 0, 0, 1);
+    static white = new Color(1, 1, 1, 1);
+    static transparent = new Color(0, 0, 0, 0);
+    static red = new Color(1, 0, 0, 1);
```

Comment:
@HarelM , this is the only logic that changed - I didn't understand why the static properties were added under

Reply:
This is probably from the time before typescript, where this was the way to achieve it using javascript.

Reply:
Looks better now, thanks!